src/Illuminate/Console/GeneratorCommand.php

```diff
@@ -157,6 +157,13 @@ public function handle()
         $this->makeDirectory($path);
 
         $this->files->put($path, $this->sortImports($this->buildClass($name)));
+        
+        event("console.generated: {$this->type}", [
```

Comment:
You can't call this foundation function. Also, this package doesn't depend on the event package, anyway.

Reply:
@GrahamCampbell ah sorry, new to the framework code.

Would calling the container be ok?
```php
Container::getInstance()
    ->make(Dispatcher::class)
    ->dispatch("console.generated: {$this->type}", [
        'name' => $name,
        'path' => $path,
        'options' => $this->option(),
        'arguments' => $this->argument(),
    ]);
```